.github/workflows/package_for_test.yml

```diff
@@ -49,17 +49,23 @@ jobs:
           git config --global core.autocrlf false
           git config --global core.eol lf
 
+      - name: Checkout for push
+        uses: actions/checkout@v3
+        if: ${{ github.event_name == 'push' }}
+
       - name: Get PR Branch
         if: github.event_name == 'issue_comment'
         id: comment-branch
-        uses: xt0rted/pull-request-comment-branch@v1
+        uses: yanguoyu/pull-request-comment-branch@v2.0.0.1
```

Comment:
Which convention is `v2.0.0.1` following

Reply:
Avoid sync `xt0rted/pull-request-comment-branch` version conflict.

Reply:
I think this is probably unnecessary, since org or user is already distinguishable.
Or since we are not going to use the standard version number anymore, we could consider v2.0.0-fixABC?

Reply:
`yanguoyu/pull-request-comment-branch@v2.0.0.1` means tag v2.0.0.1, after sync `xt0rted/pull-request-comment-branch`, it may have conflict tag name.
`2.0.0.1` means the version is same as `xt0rted/pull-request-comment-branch` 2.0.0, and only adds head_repo and head_owner metadata.

Reply:
> 2.0.0.1 means the version is same as xt0rted/pull-request-comment-branch 2.0.0, and only adds head_repo and head_owner metadata.

It sounded logical and I was convinced.

Reply:
@Keith-CY Anything else?

Reply:
> `yanguoyu/pull-request-comment-branch@v2.0.0.1` means tag v2.0.0.1, after sync `xt0rted/pull-request-comment-branch`, it may have conflict tag name. `2.0.0.1` means the version is same as `xt0rted/pull-request-comment-branch` 2.0.0, and only adds head_repo and head_owner metadata.

It should be `2.1.0` in semver convention because it's a feature that doesn't break compatibility.

Besides, the 4th character is defined(https://semver.org/#spec-item-9) in semver.

Since ours won't be released officially, I would suggest naming it `v2.1.0-beta`

Reply:
The 4th field is can set as a number. Maybe we can set it to `v2.0.0-0`?

Reply:
> The 4th field is can set as a number. Maybe we can set it to `v2.0.0-0`?

`v2.0.0-0` should be released earlier than `v2.0.0` because `-0` indicates that `v2.0.0-0` is a prerelease of `v2.0.0`. Any update adopted on `v2.0.0` should be released along with `v2.1.0-x`(https://semver.org/#spec-item-3)

`yanguoyu/pull-request-comment-branch` can start from `v2.0.0` because it's the first release, so the version number can be arbitrary. But accruing from the released `xt0rted/pull-request-comment-branch@v2.0.0` makes it clear.

Reply:
Gotcha, I will set it to `v2.1.0-beta`